grr/core/grr_response_core/config/server.py

```diff
@@ -282,6 +282,15 @@
     "If the average network usage per client becomes "
     "greater than this limit, the hunt gets stopped.")
 
+# GRRafana HTTP Server settings.
+config_lib.DEFINE_string("GRRafana.server",
```

Comment:
I'm also increasingly concerned about the name. Basically: `grrafana` and `grafana` are indistinguishable when reading the text and relying on upper casing to highlight the difference feels a bit odd. Even though I like the name, I think we should go with something conventional, like `GrafanaSourceServer` or `GrafanaDataSourceServer`. (Not for this PR, but let's have a discussion on this)